packages/modules/cart/src/migrations/Migration20241205095237.ts

```diff
@@ -50,9 +50,6 @@ export class Migration20241205095237 extends Migration {
       'CREATE INDEX IF NOT EXISTS "IDX_cart_line_item_adjustment_item_id" ON "cart_line_item_adjustment" (item_id) WHERE deleted_at IS NULL;'
     )
 
-    this.addSql(
-      'alter table if exists "cart_line_item_tax_line" alter column "rate" type integer using ("rate"::integer);'
-    )
     this.addSql(
```

Comment:
We need to remove this from previous migration to not convert existing entires into integers

Reply:
Nice catch